apps/miranum-bpmn-modeler-webview/src/main.ts

```diff
@@ -340,4 +315,51 @@ function asyncDebounce<F extends (...args: any[]) => Promise<any>>(
         }) as ReturnType<F>;
 }
 
+function createBpmnModeler(executionPlatformVersion: ExecutionPlatformVersion): Modeler {
+    let bpmnModeler;
+    const commonModules = [
+        // Token Simulation
+        TokenSimulationModule,
+    ];
+    switch (executionPlatformVersion) {
+        case ExecutionPlatformVersion.None:
+        case ExecutionPlatformVersion.Camunda7: {
+            bpmnModeler = new BpmnModeler7({
+                container: "#js-canvas",
+                keyboard: {
+                    bindTo: document,
+                },
+                propertiesPanel: {
+                    parent: "#js-properties-panel",
+                },
+                additionalModules: [
+                    ...commonModules,
+                    ElementTemplateChooserModule,
+                    miragonProviderModule,
+                ],
+            });
+            break;
+        }
+        case ExecutionPlatformVersion.Camunda8: {
+            bpmnModeler = new BpmnModeler8({
+                container: "#js-canvas",
+                keyboard: {
+                    bindTo: document,
+                },
+                propertiesPanel: {
+                    parent: "#js-properties-panel",
+                },
+                additionalModules: [
+                    ...commonModules,
+                    ElementTemplateChooserModule,
+                    // Token Simulation
+                    TokenSimulationModule,
+                ],
+            });
+            break;
+        }
+    }
+    return bpmnModeler;
```

Comment:
```suggestion
    let bpmnModeler;
    const commonModules = [
        ElementTemplateChooserModule,
        TokenSimulationModule,
    ];
    switch (executionPlatformVersion) {
        case ExecutionPlatformVersion.None:
        case ExecutionPlatformVersion.Camunda7: {
            bpmnModeler = new BpmnModeler7({
                container: "#js-canvas",
                keyboard: {
                    bindTo: document,
                },
                propertiesPanel: {
                    parent: "#js-properties-panel",
                },
                additionalModules: [
                    ...commonModules,
                    miragonProviderModule,
                ],
            });
            break;
        }
        case ExecutionPlatformVersion.Camunda8: {
            bpmnModeler = new BpmnModeler8({
                container: "#js-canvas",
                keyboard: {
                    bindTo: document,
                },
                propertiesPanel: {
                    parent: "#js-properties-panel",
                },
                additionalModules: [
                    ...commonModules,
                ],
            });
            break;
        }
    }
    return bpmnModeler;
```

Reply:
🔧 We can put the ElementTemplateChooser in the commonModules since it works for both modelers.